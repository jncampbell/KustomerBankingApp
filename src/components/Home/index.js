import React from 'react';
import MainContainer from '../MainContainer';

const Home = () => (
  <MainContainer />
);

export default Home;
