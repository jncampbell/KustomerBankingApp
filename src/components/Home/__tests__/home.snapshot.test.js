import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../index';

jest.mock('../../MainContainer', () => 'MainContainer');

describe('Home', () => {
  it('has consistent render output', () => {
    const component = renderer.create(
      <Home />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
