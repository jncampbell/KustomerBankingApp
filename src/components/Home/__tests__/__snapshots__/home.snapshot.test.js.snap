// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home has consistent render output 1`] = `<MainContainer />`;
